core/src/test/java/io/forestframework/core/http/routing/AbstractHttpIntegrationTest.java

```diff
@@ -21,7 +23,14 @@
 
     public HttpResponse sendHttpRequest(String method, String path) throws IOException {
         String uri = "http://localhost:" + port + path;
-        HttpUriRequest request = RequestBuilder.create(method).setUri(uri).build();
+        HttpUriRequest request = RequestBuilder
+                .create(method)
+                .setUri(uri)
+                .setHeader("Accept", String.valueOf(ContentType.APPLICATION_JSON))
+                .setHeader(HttpHeaders.CONTENT_TYPE, "application/json")
+//                .setHeader("User-Agent", "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/58.0.3029.110 Safari/537.36")
+//                .setHeader(HttpHeaders.CONTENT_TYPE, "text/html")
```

Comment:
Either remove the comment or uncomment, why do you leave the comments here?

Reply:
i just forgot to delete 'em. it's not a good practice.